Python/bytecodes.c

```diff
@@ -669,7 +669,7 @@ dummy_func(
             EXIT_IF(!PyFloat_CheckExact(value_o));
         }
 
-        pure op(_BINARY_OP_MULTIPLY_FLOAT, (left, right -- res)) {
+        op(_BINARY_OP_MULTIPLY_FLOAT, (left, right -- res)) {
```

Comment:
Why did you un-annotate these three `float` ops?

Reply:
They use the strange `_PyFloat_FromDouble_ConsumeInputs` which take stackrefs.